caput/memh5.py

```diff
@@ -2822,6 +2941,9 @@ def _copy_from_file(h5group, memgroup, selections=None):
     return group
 
 
+# Some extra functions for types. Should maybe move elsewhere
+
```

Comment:
I'm debating moving everything below to a new file since their utility is grouped together quite well and it adds a lot of clutter to `memh5`. Still undecided though